yellowbrick/features/manifold.py

```diff
@@ -299,179 +332,103 @@ def fit_transform(self, X, y=None):
 
         Note also that fit records the amount of time it takes to fit the
         manifold and reports that information in the visualization.
-
+        
         Parameters
         ----------
         X : array-like of shape (n, m)
-            A matrix or data frame with n instances and m features where m > 2.
+            A matrix or data frame with n instances and m features
 
         y : array-like of shape (n,), optional
             A vector or series with target values for each instance in X. This
             vector is used to determine the color of the points in X.
 
         Returns
         -------
-        self : Manifold
```

Comment:
Can we move this `returns` documentation up to `fit?`

Reply:
My bad! Will correct it!